src/base/utils.ml

```diff
@@ -1,48 +1,474 @@
 open Core_kernel
-
-(** Alias [Cvar] to avoid shadowing. *)
+module Types0 = Types
 module Cvar0 = Cvar
+module Checked_ast = Checked_ast
```

Comment:
I'm not sure this re-definition is useful. It even looks like `Checked_ast` is not used in the rest of the file.

Reply:
It probably is not, I copied most of the definitions from `snark0` when I was not sure how much I was going to port, I'll delete the unneeded ones.